src/routes/login/index.js

```diff
@@ -67,9 +68,18 @@ const Login = () => {
                 <input {...register('password', { required: true })} name="password" type="password" placeholder="••••••••••••••••••••••••" />
               </div>
               {errors?.password && t('Password can\'t be empty.')}
+              {config.medienhaus.customServer &&
+                <details>
+                  <summary>{t('Advanced')}</summary>
+                  <div>
+                    <label htmlFor="server">{t('server')}</label>
+                    <input {...register('server')} name="server" type="server" placeholder="https://matrix.org" />
+                  </div>
+                </details>}
               {serverResponseErrorMessage && <p>❗️ {serverResponseErrorMessage}</p>}
               <button name="submit" type="submit" disabled={isLoading}>{isLoading ? <Loading /> : 'LOGIN'}</button>
             </form>
+            {config.medienhaus.customServer && <p>{t('No account yet?')} <a href="https://app.element.io/#/register" rel="external nofollow noopener noreferrer" target="_blank">{t('Create Account')}</a></p>}
```

Comment:
There's already a "Register new account?" link that one can define via `.env` variables